string detector_ns
geometry_msgs/TransformStamped t1
geometry_msgs/TransformStamped t2
---
